pkg/util/validation/validation.go

```diff
@@ -144,6 +149,7 @@ func IsDNS1035Label(value string) []string {
 // - valid: *.bar.com, *.foo.bar.com
 // - invalid: *.*.bar.com, *.foo.*.com, *bar.com, f*.bar.com, *
 const wildcardDNS1123SubdomainFmt = "\\*\\." + dns1123SubdomainFmt
+const wildcardDNS1123SubdomainErrMsg = "a valid wildcard DNS (RFC 1123) subdomain label must start with '*.', followed by a valid DNS subdomain name, which must consist of lower case alphanumeric character, '-' or '.' and end with an alphanumeric character"
```

Comment:
Pluralize first `character`: `must consist of lower case alphanumeric characters, ...`